sushibar-backend-avro/src/main/resources/avro/Order.avsc

```diff
@@ -1,11 +1,28 @@
 {
-     "type": "record",
-     "namespace": "org.virtualsushibar.backend.avro",
-     "name": "Order",
-     "version": "1",
-     "fields": [
-        { "name": "orderId", "type": "string" },
-        { "name": "amount", "type": "int" },
-        { "name": "meal", "type": "string" }
-     ]
+  "type": "record",
+  "namespace": "org.virtualsushibar.backend.avro",
+  "name": "Order",
+  "version": "1",
+  "fields": [
+    {
+      "name": "orderId",
+      "type": {
+        "type": "string",
+        "avro.java.string": "String"
+
+      }
+    },
+    {
+      "name": "amount",
```

Comment:
Shall we use double or float for the amount to have some decimal precision

Reply:
it was conceived as a product identifier, such as a can of coke, a given pizza, therefore a non-decimal quantity. In the future we can think about extending it